example/main.tf

```
@@ -0,0 +1,51 @@
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  namespace = "example"
+  stage     = "dev"
+  name      = "app"
+}
+
+module "subnets" {
+  source             = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=master"
+  namespace          = "example"
+  stage              = "dev"
+  name               = "app"
+  region             = "us-west-2"
+  vpc_id             = "${module.vpc.vpc_id}"
+  igw_id             = "${module.vpc.igw_id}"
+  cidr_block         = "10.0.0.0/16"
+  availability_zones = ["us-west-2a", "us-west-2b"]
+}
+
+module "elasticsearch" {
+  source                         = "git::https://github.com/cloudposse/terraform-aws-elasticsearch.git?ref=tags/0.1.5"
+  namespace                      = "example"
+  stage                          = "dev"
+  name                           = "es"
+  dns_zone_id                    = "Z3SO0TKDDQ0RGG"
+  security_groups                = []
+  vpc_id                         = "${module.vpc.vpc_id}"
+  subnet_ids                     = ["${module.subnets.public_subnet_ids}"]
+  zone_awareness_enabled         = "true"
+  elasticsearch_version          = "6.2"
+  instance_type                  = "t2.small.elasticsearch"
+  instance_count                 = 4
+  kibana_subdomain_name          = "example-kibana-es"
+  encrypt_at_rest_enabled        = "false"
+  ebs_volume_size                = 10
+  iam_actions                    = ["es:*"]
+  iam_role_arns                  = ["*"]
+  create_iam_service_linked_role = "false"
+}
+
+module "elasticsearch-cleanup" {
```

```suggestion
module "elasticsearch_cleanup" {
```

use snake case for naming resources